backend/src/main/java/com/huellapositiva/domain/exception/InvalidStatusId.java

```diff
@@ -0,0 +1,7 @@
+package com.huellapositiva.domain.exception;
+
+public class InvalidStatusId extends RuntimeException{
```

Comment:
-

Reply:
The class was duplicated. We removed the class.